pages/testdropdown.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+        #boite-recherche {
+            box-sizing: border-box;
+            background-image: url('searchicon.png');
+            background-position: 14px 12px;
+            background-repeat: no-repeat;
+            font-size: 16px;
+            padding: 14px 20px 12px 45px;
+            border: none;
+            border-bottom: 1px solid #ddd;
+            width: 100%;
+            /* Pour occuper toute la largeur */
+        }
+
+        #boite-recherche:focus {
+            outline: 3px solid #ddd;
+        }
+
+        .dropdown {
+            position: relative;
+            display: inline-block;
+        }
+
+        .my-dropdown-content {
+            position: absolute;
+            background-color: #f6f6f6;
+            min-width: 230px;
+            overflow: auto;
+            border: 1px solid #ddd;
+            z-index: 1;
+        }
+
+        .my-dropdown-content li {
+            color: black;
+            padding: 12px 16px;
+            text-decoration: none;
+            display: block;
+            cursor: pointer;
+            /* Pour montrer que c'est cliquable */
+        }
+
+        .my-dropdown-content li:hover {
+            background-color: #ddd;
+        }
+
+        #ingredients-selectionne {
+            padding: 10px;
+            background-color: #f6f6f6;
+            border: 1px solid #ddd;
+        }
+
+        #noResult {
+            padding: 10px;
+            background-color: #f6f6f6;
+            border: 1px solid #ddd;
+        }
+
+        #ingredients-selectionne {
+            padding: 10px;
+            background-color: #f6f6f6;
+            border: 1px solid #ddd;
+            margin-top: 10px;
+            /* Ajout de marge en haut */
+        }
+
+        .ingredients-selectionne {
+            display: inline-block;
+            padding: 5px 10px;
+            background-color: #ddd;
+            border-radius: 5px;
+            margin-right: 5px;
+            /* Ajout de marge entre les ingrédients */
+            cursor: pointer;
+        }
+
+        .ingredients-selectionne:hover {
+            background-color: #aaa;
+        }
+    </style>
+</head>
+
+<body style="background-color:white;">
+
+    <section class="dropdown">
+        <div id="my-dropdown" class="my-dropdown-content">
+            <input type="text" placeholder="Ajouter un ingredient..." id="boite-recherche" onkeyup="filterFunction()">
```

Comment:
Pas de JavaScript dans les balises HTML, ajoute un `eventListener` dans un fichier séparé
```suggestion
            <input type="text" placeholder="Ajouter un ingredient..." id="boite-recherche">
```